tests/test-extensions/some-extension/src/main/java/org/mule/test/some/extension/SomeOps.java

```diff
@@ -26,8 +29,27 @@
 
   private final Logger LOGGER = getLogger(SomeOps.class);
 
+  @Parameter
+  @Optional
+  private ZonedDateTime zonedDateTime;
+
   public void someOp(@Connection String conn, @Config SomeExtension ext) {}
 
+  public SomeExtension retrieveConfiguration(@Config SomeExtension config) {
+    return config;
+  }
+
+  public ZonedDateTime retrieveOperationZonedDateTime() {
+    return zonedDateTime;
+  }
+
+  public boolean isAfter(ZonedDateTime zonedDateTimeParam, @Config SomeExtension config) {
```

Comment:
it is not clear to me the purpose of this operation it just tests the isAfter method of the Java class

Reply:
The intention of this method was to add a test scenario to validate that a ZonedDateTime operation parameter/argument is being correctly parsed from the configured string on the flow xml file.

Reply:
Maybe make the test just return the parameter then? Plus adding a parameter to the SomeOps class like you did:
```
  @Parameter
  @Optional
  private ZonedDateTime zonedDateTime;
```
 makes all methods/operations defined in that class also have that parameter already. So maybe this can be simplified so there are less parameters and it is not so confusing.

Reply:
Yes, makes sense, I will simplify the operation as you suggested